src/AccessibilityTranslator.js

```diff
@@ -15,7 +14,7 @@ export default class AccessibilityTranslator extends React.Component {
    */
   static get propTypes() {
     return {
-      stage: PropTypes.object.isRequired,
+      stage: PropTypes.object.isRequired, // eslint-disable-line
```

Comment:
Please add the particular rule(s) that need to be disabled instead of disabling all linting.  It helps others understand what the linting issue is for this line.